src/main/java/org/folio/ed/service/DcbTransactionService.java

```diff
@@ -15,8 +16,18 @@ public class DcbTransactionService {
 
   private final DcbClient dcbClient;
 
-  public ResponseEntity<TransactionStatusResponse> getDcbTransactionStatus(String dcbTransactionId) {
-    log.info("getDcbTransactionStatus:: Getting transaction status for id: {}", dcbTransactionId);
+  public TransactionStatusResponse getDcbTransactionStatusById(String dcbTransactionId) {
+    log.info("getDcbTransactionStatusById:: Getting transaction status for id: {}", dcbTransactionId);
     return dcbClient.getDcbTransactionStatus(dcbTransactionId);
   }
+
+  public TransactionStatusResponse createDCBTransaction(String dcbTransactionId, DcbTransaction dcbTransaction) {
+    log.info("createDCBTransaction:: Creating transaction for id: {}", dcbTransactionId);
+    return dcbClient.createCirculationRequest(dcbTransactionId, dcbTransaction);
```

Comment:
Is this name correct createCirculationRequest ? I would prefer method name createTransaction

Reply:
@gurleenkaurbp :- name is createDCBTransaction only ,  **createCirculationRequest** is from real dcb. 
shall i change it?

Reply:
hmm ok I believe then lets keep it but in dcb the name needs to be revisited at some point